chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -225,10 +225,10 @@ fi
 
 # for making HA on shared NFS backend storage work
 add_group glance 200
-add_user glance 200 glance
 add_group qemu 201
```

Comment:
Looks like this is mirrored in autoyast as well, and the qemu group is also there